app.css

```diff
@@ -0,0 +1,83 @@
+html {
+  box-sizing: border-box;
+}
+html,
+body {
+  font-size: 14px;
+}
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+body {
+  font-family: Sans-serif, Helvetica Neue, Arial, Helvetica;
```

Comment:
I think you have these backwards...also, `sans-serif` should be lowercase, though it might not break anything